README.md

```diff
@@ -23,6 +23,7 @@
 | PROJECTNAME          | Name of the Project.                              | NONE * |
 | VERSION              | Version of the Project.                           | NONE * |
 | LOGO                 | An image to be used as logo for the Project.      | path relative to DOCPATH. *example* - <i>To use DOCPATH/images/logo.svg as the logo, set LOGO as images/logo.svg</i>.|
+| MARKDOWN_FLAVOUR     | Input file format flavour. (You can see list of supported format [here](http://pandoc.org/MANUAL.html))                         | markdown_github   |
```

Comment:
Instead of linking to pandoc's documentation, I would suggest to just state all the possible values seperated by `|` in the format section. AFAIK there are only 4 possible values. Just list them. other than that LGTM.